migen/build/platforms/sinara/kasli_soc.py

```diff
@@ -0,0 +1,314 @@
+from migen.build.generic_platform import *
+from migen.build.xilinx import XilinxPlatform
+
+
+_io = [
+    ("clk500", 0, Pins("B24"), IOStandard("LVCMOS25")),
+
+    ("user_led", 0, Pins("AF19"), IOStandard("LVCMOS25")),
+    ("user_led", 1, Pins("AF23"), IOStandard("LVCMOS25")),
+    ("user_led", 2, Pins("A23"), IOStandard("LVCMOS25")),
+    ("error_led", 0, Pins("D20"), IOStandard("LVCMOS25")),
+
+    ("serial", 0,
+        Subsignal("tx", Pins("Y18")),
+        Subsignal("rx", Pins("AA18")),
+        IOStandard("LVCMOS25"),
+    ),
+
+    ("user_sma_clock", 0,
+        Subsignal("p", Pins("AD20"), IOStandard("LVCMOS25"),
+            Misc("DIFF_TERM=TRUE")),
+        Subsignal("n", Pins("AD21"), IOStandard("LVCMOS25"),
+            Misc("DIFF_TERM=TRUE"))
+    ),
+
+    ("i2c", 0,
+        Subsignal("scl", Pins("B22")),
+        Subsignal("sda", Pins("B20")),
+        IOStandard("LVCMOS18")
+    ),
+
+    ("clk_gtp", 0,
+        Subsignal("p", Pins("R6")),
+        Subsignal("n", Pins("R5")),
+        IOStandard("LVCMOS25")
```

Comment:
Wrong I/O standard. That would be CML I think, but it should not be specified on GTREFCLK pins anyway.